Userland/Shell/Builtin.cpp

```diff
@@ -922,8 +922,8 @@ int Shell::builtin_time(int argc, const char** argv)
         warnln("Timing report:");
         warnln("==============");
         warnln("Command:         {}", String::join(' ', args));
-        warnln("Average time:    {:.2} ms (median: {}, stddev: {:.2})", iteration_times.average(), iteration_times.median(), iteration_times.standard_deviation());
-        warnln("Excluding first: {:.2} ms (median: {}, stddev: {:.2})", iteration_times_excluding_first.average(), iteration_times_excluding_first.median(), iteration_times_excluding_first.standard_deviation());
+        warnln("Average time:    {:.2} ms (median: {}, stddev: {:.2}, min: {}, max:{})", iteration_times.average(), iteration_times.median(), iteration_times.standard_deviation(), iteration_times.min(), iteration_times.max());
+        warnln("Excluding first: {:.2} ms (median: {}, stddev: {:.2}, min: {}, max:{})", iteration_times_excluding_first.average(), iteration_times_excluding_first.median(), iteration_times_excluding_first.standard_deviation(), iteration_times_excluding_first.min(), iteration_times_excluding_first.max());
```

Comment:
Seems like this line keeps getting l o n g e r, might be worth splitting it up to multiple lines at this point

Reply:
Might even be cool is we could ask `AK::Statistics` to print out a summary?

Reply:
@mustafaquraish This is a great idea! Added to the little tweaks list. :^)